misk-jobqueue/src/main/kotlin/misk/jobqueue/JobQueue.kt

```diff
@@ -41,4 +41,69 @@ interface JobQueue {
     idempotenceKey: String = UUID.randomUUID().toString(),
     deliveryDelay: Duration? = null,
     attributes: Map<String, String> = mapOf())
+
+  /**
+   * Batch Enqueue a set of jobs onto the given queue, along with a set of job attributes.
+   *
+   * @param queueName The name of the queue on which to place the job.
```

Comment:
https://publicobject.com/2007/08/05/stop_writing_redundant_javadoc/

Reply:
removed what it think is redundant